### PR TITLE
fix: 修复顶部菜单模式，菜单布局不生效。 (fix #121)

### LINKED ISSUE #121: "顶部菜单布局"设置无效
image

### PR DESCRIPTION
**Pull request by sunHsl**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [x] Pull request template structure not broken

### `Type`

> ℹ️ What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### `Checklist`

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

> 👉 _Put an `x` in the boxes that apply._

- [x] My code follows the style guidelines of this project
- [x] Is the code format correct
- [x] Is the git submission information standard?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/layouts/src/components/menu/index.vue [hunk 1]
```diff
@@ -135,5 +143,21 @@ const routerToMenu = (item: RouteRecordItem) => {
     flex: 1;
     flex-basis: auto;
   }
+
+  & .n-menu--horizontal {
```
Comment (likui628) [inline packages/layouts/src/components/menu/index.vue:147]: n-menu--horizontal是naive-ui的，如果以后换了VbenMenu换了其它组件库就会有问题

Comment (likui628) [reply packages/layouts/src/components/menu/index.vue:147]: 根据getTopMenuAlign可以计算出相应的tailwind样式justify-start，justify-center，justify-end

Comment (sunHsl) [reply packages/layouts/src/components/menu/index.vue:147]: tailwind样式我担心其他地方没有用过的话，用js动态生成可能编译不出样式，n-menu--horizontal这个我没有太好的解决方案，只能根据不同的组件库适配。

Comment (likui628) [reply packages/layouts/src/components/menu/index.vue:147]: 那就直接计算相应的style属性就行了，不跟tailwind绑定了。
`:style="getMenuStyle"`,根据getTopMenuAlign计算就行

**Review by likui628 — APPROVED** [review]

**Pull request merged**